README.md

```diff
@@ -50,11 +50,19 @@ Confluence page (*DESC member only*).
 
 ### Extragalactic Catalogs and Add-ons
 
+#### SkySim/DiffSky
+*by Andrew Hearin, Eve Kovacs, Patricia Larsen, Esteban Rangel, Katrin Heitmann et al.*
+
+- `skysim5000_v3.1`: prototype of new SkySim catalog, produced with differentiable, forward modeling techniques, sky area (~400 sq. deg.)
```

Comment:
I believe the catalog name is `skysim_v3.1` and there's no 5000. Suggest to also make it clear that this is not SkySim5000. 

```suggestion
- `skysim_v3.1`: prototype of new SkySim (also known as DiffSky) catalog, produced with differentiable, forward modeling techniques, sky area (~400 sq. deg.). Not to be confused with SkySim5000.
```